pkg/armhelpers/azurestack/azureclient.go

```diff
@@ -14,6 +14,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/Azure/aks-engine/pkg/armhelpers"
```

Comment:
The only thing I'd say here is that this "child" package depending upon the parent can lead to circular dependencies in the future. Ideally we'd put `NewKubernetesClientSetClient` somewhere that both `pkg/armhelpers` and `pkg/armhelpers/azurestack` could import in common. Not sure if there's a candidate package for that?

Reply:
yeah, I thought about that, I was not sure if moving more stuff around was desired, I will look for a new home for this type.